contracts/VToken.sol

```diff
@@ -818,6 +1093,82 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         return actualRepayAmount;
     }
 
+    /**
+     * @dev Allows a borrower to swap his debt between stable and variable mode, or vice versa
+     * @param rateMode The rate mode that the user wants to swap to
+     **/
+    function swapBorrowRateMode(uint256 rateMode) external {
+        accrueInterest();
+
+        /* Verify market's block number equals current block number */
+        if (accrualBlockNumber != _getBlockNumber()) {
+            revert SwapBorrowRateModeFreshnessCheck();
+        }
+
+        address account = msg.sender;
+        uint256 variableDebt = _borrowBalanceStored(account);
+        uint256 stableDebt = _updateUserStableBorrowBalance(account);
+        uint256 accountBorrowsNew = variableDebt + stableDebt;
+        uint256 stableBorrowsNew;
+
+        if (InterestRateMode(rateMode) == InterestRateMode.STABLE) {
+            require(variableDebt > 0, "vToken: swapBorrowRateMode variable debt is 0");
+
+            stableBorrowsNew = stableBorrows + variableDebt;
+            uint256 stabelBorrowRate = stableBorrowRatePerBlock();
```

Comment:
Typo: `stabelBorrowRate` vs `stableBorrowRate`. It is in more places in the code

Reply:
Fixed, can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/132/commits/0f14951544955613711dd9321a85af4f96073049).